src/core/backend.cc

```diff
@@ -241,9 +242,11 @@ InferenceBackend::SetConfiguredScheduler(
   if (config_.has_sequence_batching()) {
     RETURN_IF_ERROR(SequenceBatchScheduler::Create(
         config_, runner_cnt, OnInit, OnRun, &scheduler));
-  } else {
+  } else if (config_.has_dynamic_batching()) {
```

Comment:
Dynamic batching has to be the default choice because if the model configuration does not list any scheduler then it should use the dynamic scheduler.